install-daemon.sh

```diff
@@ -113,6 +113,17 @@ function detect_distro {
 }
 
 function check_os_comp {
+  MACHINE_TYPE=$(uname -m)
+  if [ "${MACHINE_TYPE}" != "x86_64" ]; then # check the architecture
+      print_warning "Detected architecture $MACHINE_TYPE"
+	   print_warning "Using any other architecture then 64 bit(x86_64) may (and will) cause problems."
+      echo -e -n  "* Are you sure you want to proceed? (y/N):"
+	   read -r choise
```

Comment:
I believe it's spelled "choice" 🙂

Reply:
This line and the two lines above appear differently indented on GitHub. Is it just in GitHub?

Reply:
Hmmmmn I need to change my text editor. Also yeah choice another typo.